src/scripts/app.ts

```diff
@@ -129,6 +130,9 @@ export class ComfyApp {
     this.bodyLeft = $el('div.comfyui-body-left', { parent: document.body })
     this.bodyRight = $el('div.comfyui-body-right', { parent: document.body })
     this.bodyBottom = $el('div.comfyui-body-bottom', { parent: document.body })
+    this.canvasContainer = $el('div.graph-canvas-container', {
```

Comment:
This move is necessary as Vue requires teleport target to be available when the component is mounted. If we load ComfyApp after VueApp, the container need to inited in the constructor instead of setup function.